lms/djangoapps/ccx/views.py

```diff
@@ -346,6 +351,37 @@ def override_fields(parent, data, graded, earliest=None, ccx_ids_to_delete=None)
     )
 
 
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@coach_dashboard
+def update_course_details(request, course, ccx=None):
+    """
+    Update course display name for the CCX.
+    """
+    if not ccx:
+        raise Http404
+
+    # TODO: This should have been saved as a override field. This will
```

Comment:
@itsjeyd But I think that here we will need to add support to save `display_name` as an override for ccx course? So this depends on #18808

Reply:
@taranjeet That's not a genuine dependency, though? The handler gets both `course` and `ccx` passed in, so wouldn't it be possible to do something like this:

```python
    name = request.POST['display_name']
    override_field_for_ccx(ccx, course, 'display_name', name)
```

I agree that conceptually, it would make more sense to merge #18808 first. But the changes from this PR should work correctly without the changes from #18808?

Reply:
By dependency, I meant that if this PR gets merged before #18808 , then for some courses, `display_name` will be saved as an override field, while it will be missing for the existing one. But I agree that we can add the code at least.
I have updated the code snippet. Thanks :)